deploy.sh

```diff
@@ -9,10 +9,8 @@ services=(
   'uploads'
   'app-api'
   'stream-functions'
-  'ui-waflog-s3-bucket'
   'ui'
   'ui-auth'
-  'ui-waf-log-assoc'
   'ui-src'
```

Comment:
These two services were both collapsed into the 'ui' service, and no longer exist.